packages/browser/__tests__/sessionInfo/SessionInfoManager.spec.ts

```diff
@@ -20,7 +20,11 @@
  */
 
 import "reflect-metadata";
-import { mockStorageUtility } from "@inrupt/solid-client-authn-core";
+import {
+  mockStorageUtility,
+  StorageUtility,
+} from "@inrupt/solid-client-authn-core";
+import { mockStorage } from "@inrupt/solid-client-authn-core/dist/storage/__mocks__/StorageUtility";
```

Comment:
If these shouldn't be relative imports, then it's still a bit weird that one imports from the package proper, and the other reaches into `/dist`.

Reply:
Oops yeah that looks like a good ol' auto import

Reply:
Actually no, the export was missing: 0c89e2c953720663712c5ab8f014a6c72a417ce7